src/app/containers/exiu-ccd-connector-wrapper/exui-ccd-connector.component.ts

```diff
@@ -17,14 +25,15 @@ import {Store} from '@ngrx/store';
   `
 })
 export class ExuiCcdConnectorComponent implements AfterContentInit, OnDestroy {
-
   @ContentChild('ccdComponent') ccdComponent;
+  @ContentChild('ccdComponent', {read: ElementRef}) ccdComponentElementRef: ElementRef;
```

Comment:
you got double ContentChild decorator... no need to reference it twice. Think you should be able to extend line 28

Reply:
Unfortunately, we need both. We need both a reference to both the HTML element and the component instance itself.